src/crate/client/http.txt

```diff
@@ -153,7 +153,7 @@ Uploading a blob to a table with disabled blob support throws an exception::
     ...     'locations', '040f06fd774092478d450774f5ba30c5da78acc8', f)
     Traceback (most recent call last):
     ...
-    BlobsDisabledException: locations/040f06fd774092478d450774f5ba30c5da78acc8
+    ProgrammingError: 404 Client Error: Not Found
```

Comment:
@chaudum looked into it
We changed the error handling to return 404 instead of 400 if a index/shard is not found.
I think we should change https://github.com/crate/crate-python/blob/master/src/crate/client/http.py#L330